surveys/sphinx-survey-2025.yml

```diff
@@ -171,16 +179,38 @@ questions:
 
 # Motivation: Understand the constellation of tools for documentation this responder works with.
 - title: "Which Python docstring style do you use?"
+  description: >-
+    Select N/A if you don’t use Python or Python docstrings.
+
+    Google-style docstrings look like:
+    ```rst
+    Arguments:
+        my_param (type): description
+    ```
+
+    Numpydoc-style docstrings look like:
+    ```rst
+    Arguments
+    ---------
+    my_param : type
+        description
+    ```
+
+    Sphinx-style docstrings look like:
+    ```rst
+    :param [my_param]: description
+    ```
```

Comment:
cc @timhoffm I think this is a nicer way to describe/show the different docstring styles. What do you think?